chart/values.yaml

```diff
@@ -50,6 +50,8 @@ database:
   tlsMode: require
   # database.name -- Name of the fusionauth database
   name: fusionauth
+  # for Managed cloud instances, it is not expected to pass the root credentials of the database.
+  isManagedDB: false
```

Comment:
Rather than an additional setting, would it make more sense to just not try to set these if values are committed?

Reply:
Yeah, that makes absolute sense. Do you want me to amend the PR, or happy to proceed on your end?

Reply:
If you can update the PR that would be helpful, thanks!

Reply:
Updated